src/components/ContentSidebar/ActivityFeed/activity-feed/ActiveState.js

```diff
@@ -90,8 +91,8 @@ const ActiveState = ({
                                 getAvatarUrl={getAvatarUrl}
                                 // permissions are not part of task API so hard code to true
                                 permissions={{
-                                    task_edit: true,
-                                    task_delete: true
+                                    can_delete: true,
```

Comment:
We can do a little better here. If the user can't comment to begin with, they definitely can't edit or delete tasks.

Reply:
As we discussed, a task has a been made and we'll update in a future PR after deliberation.